src/Util.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+public class Util {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public static double distance(double side1, double side2) {
+        return Math.sqrt(Math.pow(side1, 2) + Math.pow(side2, 2));
+    }
+
+    public static int max(int... numbers) {
+        int max = Integer.MIN_VALUE;
+        for (int number : numbers) {
+            max = Math.max(max, number);
+        }
+        return max;
+    }
+
+    public static int min(int... numbers) {
+        int min = Integer.MAX_VALUE;
+        for (int number : numbers) {
+            min = Math.min(min, number);
+        }
+        return min;
+    }
+
+    public static int inputInt(String request) {
+        System.out.println(request);
+        return scanner.nextInt();
+    }
+
+    public static int getNumberLength(int number) {
+        int counter = 1;
+        while (number > 10) {
+            number /= 10;
+            counter++;
+        }
+        return counter;
+    }
+
+    public static double inputDouble(String request) {
+        System.out.println(request);
+        return scanner.nextDouble();
+    }
```

Comment:
put related funcs together (this for example with inputInt), like near eachother in the file